### DISCUSSION TITLE
plc2无法打开

### DISCUSSION THREAD

**Discussion by ppx190** [Meloong-Git/PCL#2757] [提问] [answered]
plc双击后无响应，但是鼠标会变成圆圈在转

**Comment by WForst-Breeze** [comment]
可能是电脑的问题……？🤔

**Reply by ppx190** [reply]
玩ttf都120fps，就pcl一直打不开{哭}

**Reply by ppx190** [reply]
而且圈圈就转了一秒
就没了

**Comment by kong-que-dnf123** [comment]
转圈是表示“正在加载”......
然后如果一直在转圈可能就是电脑太卡啥的（）

**Reply by ppx190** [reply]
玩ttf都120fps，就pcl一直打不开

**Reply by ppx190** [reply]
而且圈圈就转了一秒
就没了

**Reply by kong-que-dnf123** [reply]
_重新安装PCL呢（？）_

**Reply by MoYuan-CN** [reply]
重装下？
或者确认下 .Net Framework 4.6.2 （[点此下载](https://support.microsoft.com/zh-cn/topic/-net-framework-4-6-2-%E8%84%B1%E6%9C%BA%E5%AE%89%E8%A3%85%E7%A8%8B%E5%BA%8Fwindows-9dce3874-a9e5-9b11-289d-5594824aafe0)）装了？

**Reply by ppx190** [reply]
之前可以玩，但是下了国际服的瓦罗兰特以后就打不开了。

**Comment by DreamUniverse843** [comment]
是不是被 Defender SmartScreen 卡住了？
有在用系统自带的 Defender 防病毒吗？

**Reply by ppx190** [reply]
可能是，我把他删了试试

**Reply by DreamUniverse843** [reply, marked as answer]
直接删不一定能删掉...
你装个第三方防病毒软件（像是火绒啥的）给它挤掉就行了

**Reply by ppx190** [reply]
还是没用，过几天重新装一下系统吧（裂开）
